src/db/timesheets/FrontendConversions.ts

```diff
@@ -0,0 +1,102 @@
+import * as dbTypes from '../schemas/Timesheet'
+import * as frontendRowTypes from '../frontend/RowSchema'
+import * as frontendTimesheetTypes from '../frontend/TimesheetSchema'
+export class DBToFrontend {
```

Comment:
Nice! I like these conversion classes. should make it really clean and organized to have separation of our data models